app/Http/Controllers/Legacy/Web/CampaignIdsController.php

```diff
@@ -76,13 +77,18 @@ public function store(Request $request)
     }
 
     /**
-     * Show a specific campaign page.
+     * Show a specific campaign page and it's actions.
```

Comment:
😬 "its"
(I am so sorry)

Reply:
omg don't be sorry thank you for catching this! How embarrassing!